app/src/main/java/swati4star/createpdf/util/StringUtils.java

```diff
@@ -68,7 +71,14 @@ public void hideKeyboard(Activity activity) {
     }
 
     public String getDefaultStorageLocation() {
-        return Environment.getExternalStorageDirectory().getAbsolutePath() +
-                pdfDirectory;
+        File dir = new File(Environment.getExternalStorageDirectory().getAbsolutePath()
+                + PATH_SEPERATOR + pdfDirectory + PATH_SEPERATOR);
```

Comment:
The File class has a two argument constructor for files/folders in a parent folder which you could use here:
https://developer.android.com/reference/java/io/File#File(java.lang.String,%20java.lang.String)